test/source/tests/flaky.ts

```diff
@@ -107,8 +107,10 @@ export const defineFlakyTests = (testVariant: TestVariant, testWithBrowser: Test
       await SettingsPageRecipe.toggleScreen(settingsPage, 'additional');
       const fingerprint = (await settingsPage.read('.good', true)).split(' ').join('');
       const myKeyFrame = await browser.newPage(t, `chrome/settings/modules/my_key.htm?placement=settings&parentTabId=60%3A0&acctEmail=${acctEmail}&fingerprint=${fingerprint}`);
-      const files = await myKeyFrame.awaitDownloadTriggeredByClicking('@action-download-prv');
-      const key = await KeyUtil.parse(Object.values(files).pop()!.toString());
+      const downloadedFiles = await myKeyFrame.awaitDownloadTriggeredByClicking('@action-download-prv');
+      // It is not possible to have predictable file name here, because key is generated
```

Comment:
however, we have the new key's fingerprint in the `fingerprint` variable?

Reply:
file name contains not fingerprint, but key id. is it possible to compute key id from the fingerprint?

Reply:
they are the same thing

Reply:
fingerprint is longer then key id.

Reply:
In this codebase we refer to primary key id (fingerprint) as `.id` eg `key.id` which is what I used for the filename. It should be the same as the fingerprint above.

Reply:
> fingerprint is longer than key id. is key id part of fingerprint?

there may be spacing between groups of 4 characters. Why don't you print the results of the test? They should show both actual and expected values.